Formula/ipinfo-cli.rb

```diff
@@ -0,0 +1,21 @@
+class IpinfoCli < Formula
+  desc "Official CLI for the IPinfo IP Address API"
+  homepage "https://ipinfo.io/"
+  url "https://github.com/ipinfo/cli/archive/ipinfo-1.1.2.tar.gz"
+  sha256 "4172c686b15d1a92f39b3490eea4f56325d7b5f9e6696c1d12c4aedc9706a2d6"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  conflicts_with "ipinfo",
+    because: "homebrew/core/ipinfo conflicts with the binary name provided by this formula"
```

Comment:
```suggestion
  conflicts_with "ipinfo", because: "ipinfo and ipinfo-cli install the same binaries"
```
Let's try to make this consistent with other formulae.

Reply:
Thanks for the suggestion! But doesn't this possibly imply that `ipinfo` and `ipinfo-cli` are actually the same *programs*? The same "binaries" comes off to me to mean that the programs do the same thing, when here the actual issue is the same *name* only; they are very different programs.

Reply:
This is the wording used throughout the project to mean "they install a binary in the exact same location". If it was actually the same binary it would not be accepted.

Reply:
Ok, how about
```
ipinfo and ipinfo-cli install binaries of the same name
```
?

Reply:
@SMillerDev Ah I see, I'm only speaking from the eyes of an end-user, which is who I believe would see this and be confused.

@carlocab if this is alright with @SMillerDev then that looks good to me.

Reply:
If it means anything `install the same binaries` is pretty common in Homebrew/core and this is the first I've heard of it being confusing.

Reply:
Alright, if Homebrew end users have been used to this meaning of the phrase, then it's no problem for me.